arch/arm/boot/dts/Makefile

```diff
@@ -640,6 +640,10 @@ dtb-$(CONFIG_SOC_IMX6UL) += \
 	imx6ul-pico-hobbit.dtb \
 	imx6ul-pico-pi.dtb \
 	imx6ul-phytec-segin-ff-rdk-nand.dtb \
+	imx6ul-ts4100.dtb \
+	imx6ul-ts4100-8.dtb \
+	imx6ul-ts4100-7.dtb \
```

Comment:
Does order matter here? E.g. 
imx6ul-xxxx.dtb
imx6ul-xxxx-7.dtb
imx6ul-xxxx-8.dtb
imx6ul-xxxx-16.dtb
versus how it is now?
Also, what is the purpose of the ts4100 having four .dtb's?

Reply:
Order doesn't matter, that was just a typo it would seem.